docs/dm/attestation-results/attestation-results.cddl

```diff
@@ -0,0 +1,43 @@
+start = attestation-results
+
+$ar-status /= ar.status-failure
+$ar-status /= ar.status-success
+$ar-status /= ar.status-unknown
+
+attestation-results = {
+  ar.status => $ar-status
+  ar.trust-vector => ar-trust-vector
+  ? ar.raw-evidence => ar-bytes
+  ar.timestamp => ar-timestamp
+  ? ar.endorsed-claims => ar-endorsed-claims
+}
+
+ar-trust-vector = {
+  ar.hw-authenticity => $ar-status
```

Comment:
I can see this status working for most scenarios. Is there a case where values may be needed instead e.g. a certification level, an overall SW security epoc? Appreciate that those could be in the extension bucket but wanted to raise the query in case these results should be named ...-status for clarity there

Reply:
Scratch that, just read the end of the file has the -details sections!